skyportal/handlers/api/telescope.py

```diff
@@ -135,23 +135,31 @@ def get(self, telescope_id=None):
         telescopes = []
         for telescope in data:
             temp = telescope.to_dict()
-            if telescope.next_twilight_morning_astronomical() is not None:
-                temp['is_night_astronomical'] = bool(
-                    telescope.next_twilight_morning_astronomical().jd
-                    < telescope.next_twilight_evening_astronomical().jd
-                )
-                temp[
-                    'next_twilight_morning_astronomical'
-                ] = telescope.next_twilight_morning_astronomical().iso
-                temp[
-                    'next_twilight_evening_astronomical'
-                ] = telescope.next_twilight_evening_astronomical().iso
-            else:
-                temp['is_night_astronomical'] = False
-                temp['next_twilight_morning_astronomical'] = False
-                temp['next_twilight_evening_astronomical'] = False
-
+            morning = False
+            evening = False
+            is_night_astronomical = False
+            if (
+                telescope.fixed_location
+                and telescope.lon
+                and telescope.lat
+                and telescope.elevation
```

Comment:
if any of these numbers are correct and assigned to 0.0 then this statement will fail. I bet people will put in elevation = 0 if they dont know it. I think you mean:
```suggestion
                and telescope.lon is not None
                and telescope.lat is not None
                and telescope.elevation is not None
```

Reply:
Then should I check if it is not None `and` not 0.0 @profjsb ?